### DISCUSSION TITLE
اضافه کردن تاپیک

### DISCUSSION THREAD

**Discussion by ghost** [softwaretalks/newsletter#136] [General]
بد نیست چند تاپیک به ریپو اضافه کنید که تو بخش Topics دیده بشه.
تاپیک‌های پیشنهادی: computer | newsletter | software | persian | iran

ضمناً برای اضافه کردن ریپو به لیست made-in-iran، [پول ریکوئست](https://github.com/mohebifar/made-in-iran/pull/207) دادم. 
شاید لیست‌های دیگه‌ای هم باشه.

**Comment by ohmydevops** [comment]
ممنون از این پیشنهاد خوبت.
الان تاپیک‌ها رو اضافه میکنم.

**Reply by ohmydevops** [reply]
من این رو هم الان دیدم و براشون یک PR میفرستم:
https://github.com/zudochkin/awesome-newsletters
